readme.md

```diff
@@ -2,3 +2,43 @@
 
 [![Bitrise](https://app.bitrise.io/app/63d52d847cdb36db/status.svg?token=DDdEMfpVR8emhdGSgToskA&branch=master)](https://www.bitrise.io/app/63d52d847cdb36db)
 ![codecov](https://codecov.io/gh/mapbox/mapbox-events-ios/branch/master/graph/badge.svg)
+
```

Comment:
We might include a short description of what the library does... Something similar to what we have on the [android repo](https://github.com/mapbox/mapbox-events-android#about-libtelemetry) or on our [telemetry page](https://www.mapbox.com/telemetry).

Reply:
Included a brief description with a link to the telemetry page